igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -444,11 +444,11 @@ private Optional<String> getAndCacheJobLog(int jobId) {
         travisCache.setJobLog(groupKey, jobId, v3Log.getContent());
         return Optional.of(v3Log.getContent());
       }
-    } catch (RetrofitError e) {
-      if (e.getBody() != null) {
+    } catch (SpinnakerServerException e) {
+      if (e instanceof SpinnakerHttpException) { // only SpinnakerHttpException has a response body
         try {
```

Comment:
Can we get rid of this try/catch block now that we're no long called e.getBodyAs?